app/cafe.py

```diff
@@ -0,0 +1,22 @@
+import datetime
+from typing import Union
+from app.errors import (
+    NotWearingMaskError,
+    OutdatedVaccineError,
+    NotVaccinatedError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> Union[str, None]:
+        if not visitor.get("vaccine"):
+            raise NotVaccinatedError("The visitor is not vaccinated")
+        if datetime.date.today() >\
+                visitor.get("vaccine", {}).get("expiration_date"):
```

Comment:
Compare datetime.date() and None is not possible

Reply:
if this line of code **visitor.get("vaccine", {}).get("expiration_date"):** returns None, the line above, (16th line) raises an exception **NotVaccinatedError**, and this comparison will never be proceeded. Isn't it?